contracts/cw20-base/src/contract.rs

```diff
@@ -1265,6 +1270,73 @@ mod tests {
         );
     }
 
+    mod migration {
+        use super::*;
+
+        use cosmwasm_std::Empty;
+        use cw_multi_test::{App, Contract, ContractWrapper, Executor};
+
+        fn cw20_contract() -> Box<dyn Contract<Empty>> {
+            let contract = ContractWrapper::new(
+                crate::contract::execute,
+                crate::contract::instantiate,
+                crate::contract::query,
+            )
+            .with_migrate(crate::contract::migrate);
+            Box::new(contract)
+        }
+
+        #[test]
+        fn test_migrate() {
+            let mut app = App::default();
+
+            let cw20_id = app.store_code(cw20_contract());
+            let cw20_addr = app
+                .instantiate_contract(
+                    cw20_id,
+                    Addr::unchecked("meow"),
```

Comment:
Sure, this is just a one of many tests. But this is still an open source code that will be used in multiple projectes - please replace those random strings with something proper.

Reply:
Sure, np.